docs_website/docs/integrations/add_table_upload.md

```diff
@@ -0,0 +1,41 @@
+---
+id: add_table_upload
+title: Table Upload
+sidebar_label: Table Upload
+---
+
+:::warning
+This is an experimental feature with no plugin support. You can modify the source code to include custom behavior
+but they may break in future updates.
+:::
+
+## Overview
+
+Users can now upload their local CSV files or Querybook query execution results to a SQL table. This is supported by loading the
```

Comment:
upload local CSV files or use a Querybook query execution results to create a SQL table.